packages/bootstrap/middleware.js

```diff
@@ -1,14 +1,28 @@
-import { actions as dataFetchActions } from '@bufferapp/async-data-fetch';
-import { actionTypes } from '@bufferapp/async-data-fetch/src';
+import { actionTypes as dataFetchActions } from '@bufferapp/async-data-fetch';
 
 export default ({ dispatch }) => next => action => {
   next(action);
   switch (action.type) {
     case 'APP_INIT':
-      // trigger user and profiles initialization
-      dispatch({ type: 'USER_INIT' });
-      dispatch({ type: 'PROFILES_INIT' });
+      dispatch({ type: 'INIT_MODALS' });
+      dispatch({ type: 'INIT_USER' });
+      dispatch({ type: 'INIT_ORGANIZATIONS' });
+      dispatch({ type: 'INIT_APPSHELL' });
+      dispatch({ type: 'INIT_FEATURES' });
+      dispatch({ type: 'INIT_PUSHER' });
+      dispatch({ type: 'INIT_CHECK_BOOKMARKLET' });
+      dispatch({ type: 'INIT_STRIPE_DETAILS' });
       break;
+    case `organizations_${dataFetchActions.FETCH_SUCCESS}`: {
```

Comment:
could you change this to use the ORGANIZATIONS_INITIALIZED method please?
```suggestion
    case 'ORGANIZATIONS_INITIALIZED': {
```